src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage/product.html.twig

```diff
@@ -346,8 +346,8 @@
 "files recommandations": "Recommended size 800 x 800px for default theme."|trans({}, 'Admin.Catalog.Feature'),
 "files recommandations2": "JPG, GIF or PNG format."|trans({}, 'Admin.Catalog.Feature'),
 "Cover": "Cover"|trans({}, 'Admin.Catalog.Feature'),
-"Are you sure to delete this?": "Are you sure to delete this?"|trans({}, 'Admin.Notifications.Warning'),
-"This will delete the specific price. Do you wish to proceed?": "This will delete the specific price. Do you wish to proceed?"|trans({}, 'Admin.Catalog.Notification'),
+"Are you sure you want to delete this item?": "Are you sure you want to delete this item?"|trans({}, 'Admin.Notifications.Warning'),
+"Are you sure you want to delete this item?": "Are you sure you want to delete this item?"|trans({}, 'Admin.Catalog.Notification'),
```

Comment:
```suggestion
"Are you sure you want to delete this item?": "Are you sure you want to delete this item?"|trans({}, 'Admin.Notifications.Warning'),
"Are you sure you want to delete this item?": "Are you sure you want to delete this item?"|trans({}, 'Admin.Catalog.Notification'),
```
Duplicate key ? Where is the good catalog ?

Reply:
According to @LouiseBonnard [comment](https://github.com/PrestaShop/PrestaShop/issues/20632#issuecomment-679230034) I think it's Admin.Notifications.Warning

Reply:
Affirmative ;)